transplant_guide.md

```diff
@@ -120,29 +124,38 @@ Known as the most LGBT friendly neighborhood. Tons of cool shops and restaurants
     Not known for its food, but for its shows! Every Friday and Saturday Julia’s hosts Le Faux Show, a drag show of “outrageous celebrity impersonations.” Sunday brunch is taken over by a never-repeated drag show hosted by Kristie Champagne.
 
 **South Lake Union (SLU)**  
-Home of Amazon. Bro-town. Basically where the Amazon-ians go to breed. Feels like a Seattle neighborhood that was bleached years ago. 
+Home of Amazon. Bro-town. Basically where the Amazon-ians go to breed. Not a super pedestrian friendly area, and though more retail and restaurants are popping up, it doesn't feel like much of a neighborhood. A lot of the housing in this area is both really nice and really expensive, as Amazonians who want to live close to work have the salaries to afford some really nice places, and it is very central and close to downtown. Traffic can sometimes be a nightmare here because there are only a few arterial streets(it is sandwiched between I-5 and Highway 99).
```

Comment:
The traffic is also terrible because of the ongoing construction, which is only going to get worse in the short term. :)